files/porter_bootstrap

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -ex
+#!/bin/bash -e
```

Comment:
@apratina your commit message implies every porter user uses splunk, which isn't the case